runbotics/rush.json

```diff
@@ -359,28 +359,31 @@
    */
     "projects": [
         {
-            "packageName": "runbotics-orchestrator-ui",
-            "projectFolder": "runbotics-orchestrator-ui"
-        },
-        {
-            "packageName": "runbotics-desktop",
-            "projectFolder": "runbotics-desktop"
+            "packageName": "runbotics-actions-windows",
+            "projectFolder": "runbotics-actions-windows"
         },
         {
             "packageName": "runbotics-common",
             "projectFolder": "runbotics-common"
         },
         {
-            "packageName": "runbotics-actions-windows",
-            "projectFolder": "runbotics-actions-windows"
+            "packageName": "runbotics-desktop",
+            "projectFolder": "runbotics-desktop",
+            "shouldPublish": true
         },
         {
-            "packageName": "runbotics-sdk",
-            "projectFolder": "runbotics-sdk"
+            "packageName": "runbotics-orchestrator-ui",
+            "projectFolder": "runbotics-orchestrator-ui",
+            "shouldPublish": true

```

Comment:
are u sure about `shouldPublish`?

Reply:
Refering to docs: 
    //    * A flag indicating that changes to this project will be published to npm, which affects
    //    * the Rush change and publish workflows. The default value is false.
    //    * NOTE: "versionPolicyName" and "shouldPublish" are alternatives; you cannot specify them both.

Reply:
yep, its not only connected to npm publish but affects commands like `rush change` and `rush version`